GoogleDataTransport/GDTCORTests/Unit/GDTCORPlatformTest.m

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <GoogleDataTransport/GDTCORPlatform.h>
+#import <GoogleDataTransport/GDTCORReachability.h>
+
+#import "GDTCORTests/Unit/GDTCORTestCase.h"
+
+@interface GDTCORPlatformTest : GDTCORTestCase
+
+@end
+
+@implementation GDTCORPlatformTest
+
+/** Tests the reachability of mobile network connection in current platform. */
+- (void)testMobileConnectionReachability {
+  SCNetworkReachabilityFlags reachabilityFlags = [GDTCORReachability currentFlags];
```

Comment:
wrap `[GDTCORReachability currentFlags]` in XCTAssertNoThrow and then do something like XCTAssertNotEqual(reachabilityFlags, 0) afterwards.